### ISSUE TITLE
IMA interface can't be localized

### ISSUE THREAD

**Issue by icunioli** [kaltura/kaltura-player-js#204]
##### Prerequisites
- [x] Have you checked for duplicate [issues](https://github.com/kaltura/kaltura-player-js/issues)
- Which Plugin [version](https://github.com/kaltura/kaltura-player-js/releases) are you using: 0.37.3
- [x] Can you reproduce the issue with our latest release version
- [x] Can you reproduce the issue with the latest code from master
- What browser and OS names and versions are you using:
  - Chrome 71.0.3578.98 | macOS 10.14.2

##### Expected behavior
IMA SDK localizes its interface based on the UI locale set on the video player configuration.

##### Actual behavior
IMA SDK always displays its interface in English.

**Comment by kaltura-hooks** [comment]
Hi @icunioli,

Thank for you reporting an issue and helping improve Kaltura!

To get the fastest response time, and help the maintainers review and test your reported issues or suggestions, please ensure that your issue includes the following (please comment with more info if you have not included all this info in your original issue):
 - Is the issue you're experiencing consistent and across platforms? or does it only happens on certain conditions?
    please provide as much details as possible.
 - Which Kaltura deployment you're using: Kaltura SaaS, or self-hosted?
	If self hosted, are you using the RPM or deb install?
 - Packages installed.
	When using RPM, paste the output for:
```
	# rpm -qa \"kaltura*\"
```	
	For deb based systems:
```	
	# dpkg -l \"kaltura-*\"
```
 - If running a self hosted ENV - provide the MySQL server version used
 - If running a self hosted ENV - is this a single all in 1 server or a cluster?
 - If running a self hosted ENV, while making the problematic request, run:
```
	# tail -f /opt/kaltura/log/*.log /opt/kaltura/log/batch/*.log | grep -A 1 -B 1 --color \"ERR:\|PHP\|trace\|CRIT\|\[error\]\"
```
   and paste the output.
 - When relevant, provide any screenshots or screen recordings showing the issue you're experiencing.

For general troubleshooting see:
https://github.com/kaltura/platform-install-packages/blob/Jupiter-10.13.0/doc/kaltura-packages-faq.md#troubleshooting-help

If you only have a general question rather than a bug report, please close this issue and post at:
http://forum.kaltura.org

Thank you in advance,

**Comment by OrenMe** [comment]
thanks for the input @icunioli .
I've added this to our next sprint, so this should be due at end of February.
Internal tacking ID: FEC-8853

**Comment by icunioli** [comment]
Thanks @OrenMe! If there's anything I can help you with, please let me know 😃

**Comment by OrenMe** [comment]
Hi @icunioli, we are going to expose the IMA adsRendering setLocale method as a config option on the IMA plugin so you can choose it according to your site/user locale.
The fix will be up on Feb 7th.
Docs will be updated on IMA plugin repo but in a nutshell, we added a locale field to the IMA config.

**Comment by icunioli** [comment]
@OrenMe, thanks for the update!

**Comment by OrenMe** [comment]
Hi @icunioli the feature has been merged to master and released as part of 7.6 version, which is latest in production now.
See plugin params for info: https://github.com/kaltura/playkit-js-ima/blob/master/docs/api.md#parameters-2

**Comment by icunioli** [comment]
Thanks a lot for your help @OrenMe!
